_rules/SC1-3-1-aria-required-context-role.md

```diff
@@ -0,0 +1,216 @@
+---
+id: ff89c9
+name: ARIA required context role
+description: | 
+  This rule checks that a role does not exist outside of its required context.
+
+success_criterion:
+- 1.3.1 # Info and Relationships
+
+test_aspects: 
+- DOM Tree
+- CSS Styling
+
+authors:
+- Anne Thyme Nørregaard
+---
+
+## Test procedure
+
+### Applicability
+
+The rule applies to any HTML or SVG element that is [included in the accessibility tree](#included-in-the-accessibility-tree) and has an explicit [semantic role](#semantic-role) with a [WAI-ARIA required context role](https://www.w3.org/TR/wai-aria-1.1/#scope), except if the element has an implicit semantic role that is identical to its explicit semantic role.
```

Comment:
Editorial: Add an example to clarify the last point:

> except if the element has an implicit semantic role that is identical to its explicit semantic role (e.g. `<li role="listitem">`).

Reply:
Example added, but as a note to keep normative and informative parts of the rule seperate.